README.md

```diff
@@ -0,0 +1,22 @@
+# BLDC Controller
+
+This board is designed for low-speed, high torque motor applications that require
+highly precise positional and torque commands.
+
+## Features
+
+ * High Precision Current Readings
+ 
+ * 12-bit Precision Positional Readings [AS5047U-HTSM](https://ams.com/documents/20143/36005/AS5047U_DS000637_1-00.pdf)
+ 
+ * Thermal Sensitivity
+ 
+ * Chainable Initialization. Input/output connectors and 'disco-bus' pins provide
```

Comment:
Maybe this should go first in the features list since I feel like its one of the more eye-catching features.